core/DataFiles/cacert.pem

```diff
@@ -1,14 +1,13 @@
 ## How to update this file
 ## - download latest from https://curl.haxx.se/ca/cacert.pem
 ## - Paste the content below this comment
-## - Leave in there also the Let's encrypt certificate (the first one in the list)
```

Comment:
Was it on purpose to remove the let's encrypt certificate?

Reply:
As far as I know the let's encrypt CA is already trusted by every major CA so there should be no reason to additionally add it.

Reply:
If that's the case, it should be good to merge.

Reply:
I guess if this breaks anything (which it shouldn't unless something is already broken) is something we will only find out after many users are testing the beta.

Reply:
LE is crosssigned by IdenTrust which is trusted by everything newer than windows XP >SP3.